techniques/general/G223.html

```diff
@@ -47,81 +47,81 @@
+<h1><span>テクニック G223:</span>コンテンツ制作者が提供した視認性の高いフォーカスインジケータを用いる</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
-  <p>This technique is not referenced from any Understanding document.</p>
+  <p>このテクニックはどの解説書からも参照されていない。</p>
 
 
 
-  <p>This technique applies to all technologies.</p>
+  <p>このテクニックは全ての技術に適用される。</p>
 
 </div>
 </section>
 	
 
-<section id="description"><h2>Description</h2>
-    <p>The objective of this technique is enhance the focus indicator in the browser, by creating a highly visible one in the content. This technique is very similar to <a href="G195">G195</a>, but with a stronger indicator to meet <a href="https://w3c.github.io/wcag/understanding/focus-appearance">Focus Appearance</a>.</p>
+<section id="description"><h2>解説</h2>
+    <p>このテクニックの目的は、コンテンツ内に視認性の高いフォーカスインジケータを作成することによって、ブラウザのフォーカスインジケータの視認性を高めることである。このテクニックは <a href="G195">G195</a> と非常に似ているが、<a href="https://waic.jp/translations/WCAG22/Understanding/focus-appearance">フォーカスの外観</a>を満たすために、より目立つインジケータを備えている。</p>
  </section>
  <section id="examples">
-    <h2>Examples</h2>
+    <h2>事例</h2>
     <section class="example" id="example-1-links">
-       <h3>Example 1: Links</h3>
-          <p>A web page has a dark background color and light text and links. When focus lands on a link, the link is outlined with a bright yellow line, 3 pixels wide.</p>
+       <h3>事例 1: リンク</h3>
+          <p>ウェブページの背景色は暗く、テキスト及びリンクは明るい色である。リンクにフォーカスが当たると、リンクは 3 ピクセル幅の明るい黄色の線でアウトライン表示される。</p>
     </section>
     <section class="example" id="example-2-form-elements">
-       <h3>Example 2: Form Elements</h3>
+       <h3>事例 2: フォーム要素</h3>
        
-          <p>A web page includes a form inside a table. The borders of both the table and the form elements are thin, black lines. When focus lands on a form element, the element is outlined with a 5 pixel red line that is partially transparent. The red is equivalent to a hex color of #CA0000, providing a 6:1 contrast ratio with the white background.</p>
+          <p>あるウェブページには、テーブルの中にフォームが含まれている。テーブルとフォーム要素のボーダーはどちらも細い黒線である。フォーカスがフォーム要素に移動すると、その要素は 5 ピクセルの赤い線でアウトライン表示され、部分的に透明になる。この赤は 16 進数で #CA0000 の色に相当し、白い背景とのコントラスト比は 6:1 である。</p>
        
     </section>
     <section class="example" id="example-3-menus">
-       <h3>Example 3: Menus</h3>
+       <h3>事例 3: メニュー</h3>
        
-          <p>A web page includes an interactive menu with sub-menus. A user can move focus in the menu using the arrow keys. As focus moves, the currently focused menu item changes its background to a different color, which has a 15:1 contrast ratio with the surrounding items (whilst maintaining text contrast).</p>
+          <p>ウェブページには、サブメニューを含むインタラクティブなメニューが含まれている。利用者は矢印キーを使ってメニュー内でフォーカスを移動できる。フォーカスが移動すると、現在フォーカスされているメニュー項目の背景が異なる色に変わり、周囲の項目とのコントラスト比が 15:1 になる (テキストのコントラストは維持される)。</p>
```

Comment:
こちらもG195に揃えるならこんな感じでしょうか。
G195の方では「項目はその背景を異なる色に変える」となっていますが、そこは「項目の背景が異なる色に変わる」でいいような気がします。

```suggestion
          <p>サブメニューを持ったインタラクティブなメニューを含んだウェブページがある。利用者は矢印キーを使ってメニューの中でフォーカスを動かすことができる。フォーカスが動くと、フォーカスが当たっているメニュー項目の背景が異なる色に変わり、その色は周りの項目に対して 15:1 のコントラスト比になる (テキストのコントラストは維持される)。</p>
```